howso/distances.amlg

```diff
@@ -149,7 +149,7 @@
 		(accum (assoc
 			output
 				(assoc
-					distance_ratio
+					"distance_ratio"
```

Comment:
the quotes are because this is an output dictionary for the user, not declaring variables on the stack.  That's my preferred method of denoting the difference, an assoc with non-quoted keys are variables, quoted keys are simply keys for some data. Also it makes it consistent with the other detail outputs, this code was not consistent before. same for the others.